perch/views.py

```diff
@@ -229,8 +229,6 @@ def reference_links(doc):
     """Get reference links"""
     if doc.get('type') == 'organisation':
         for asset_id_type, link in doc.get('reference_links', {}).items():
-            # TODO: The API expects this data structure, but we should change
-            # it because there is always only one entry in "links".
             value = {
```

Comment:
but the result for this view will always have only one of them because we are looping over them one by one